components/libgtkhtml/Makefile

```diff
@@ -1,51 +0,0 @@
-#
```

Comment:
You can't just remove component - packages which it ships should be properly marked 'obsolete'. 
Typically you should just add line with FMRI of 'obsolete' package  to `components/meta-packages/history/history`.
For libgtkhtml this will be 
```library/desktop/libgtkhtml@2.11.1,5.11-2018.0.0.1``` , for libwnck - ```library/desktop/libwnck@2.30.7,5.11-2018.0.0.4```

Reply:
I fixed it. Thanks.